pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -877,6 +877,10 @@ bool GSRendererHW::IsSplitTextureShuffle()
 	// X might be offset by up to -8/+8, but either the position or UV should be aligned.
 	GSVector4i aligned_rc = pos_rc.min_i32(tex_rc).blend32<12>(pos_rc.max_i32(tex_rc));
 
+	// Different channel being shuffled, so needs to be handled separately (misdetection in 50 Cent)
```

Comment:
Maybe swap this with where frame_psm is set as an optimization?

Reply:
I could put it further up, yeah.